tokio-trace/benches/subscriber.rs

```diff
@@ -57,9 +57,9 @@ impl tokio_trace::Subscriber for EnabledSubscriber {
 }
 
 /// Simulates a subscriber that caches span data.
-struct AddData(Mutex<Option<span::Data>>);
+struct AddAttributes(Mutex<Option<span::Attributes>>);
```

Comment:
I like the new name!

Reply:
Yeah, `core::span::Data` was replaced with `core::span::Attributes` in 1711eaa8b284eceec7da6c63b57f33e014e8010c (when I removed the fields vector from Attributes), but the `Data` type lived on in `tokio-trace`. Since it no longer makes sense in a post-`Record` world, `tokio_trace::span::Data` was removed & the benchmark had to be updated.

Reply:
D'oh, my bad. Still!